doc/buip-hf.md

```diff
@@ -0,0 +1,103 @@
+BUIP-HF Development Notes
+===========================
+
+Specification
+----------------
+
+There is a draft specification currently located at
+
+https://github.com/BitcoinUnlimited/BUIP/blob/master/BUIP-UAHF/buip-uahf-technical-spec.md
```

Comment:
you are right, cause we renamed the dir, this is the correct one:

https://github.com/BitcoinUnlimited/BUIP/blob/master/BUIP-HF/buip-hf-technical-spec.md